### DISCUSSION TITLE
How do I use different dictionaries for different input languages?

### DISCUSSION THREAD

**Discussion by MohamedKhalil8801** [psoukie/zipchord#219] [Q&A]
I use two keyboard language layouts and switch between them regularly, the English dictionaries interfere with the other language, is it possible to have different dictionaries for each language?

**Comment by quetzal24** [comment]
I reposted a similar question (and another) 5 minutes ago, as this one remains unanswered.

**Comment by psoukie** [comment]
Having ZipChord switch dictionaries automatically with keyboard changes is on my roadmap. Unfortunately, my day job has been extremely demanding for the past months, so I was not able to progress, but I am still planning to resume the development.
